### ISSUE TITLE
Fid metrics

### ISSUE THREAD

**Issue by molu-ggg** [JulianGuerreiro/LayoutFlow#5]
Dear author,

I would like to conduct training and testing on the dataset I created by myself. However, I've encountered some problems. Your algorithm relies on the FID (Frechet Inception Distance) metric, and it seems that it needs to load a pre-trained model. There are only models trained on specified datasets like RICO and PublayNet. I traced back to previous papers through your paper, but it ultimately pointed to a paper without publicly available code. So, how should I solve this problem? 

musig = torch.load(f'./pretrained/FIDNet_musig_val_{dataset.lower()}.pt', map_location='cpu'）

**Comment by JulianGuerreiro** [comment]
Hi,
Thank you for your question. If you want to calculate the FID for your own dataset, you will unfortunately need to train your own FID network. I mention some details about that in issue #4, but you can find an example here: https://github.com/CyberAgentAILab/layout-dm/blob/main/src/trainer/trainer/fid/train.py.
